experimental/cloudstrapper/playbooks/roles/control/vars/main.yaml

```diff
@@ -0,0 +1,17 @@
+---
+
+dirTerraform: "{{ dirExpHome }}/{{ orc8rClusterName }}/terraform"
+localTerraformDir: "{{ dirTerraform }}/.terraform/modules/orc8r/orc8r/cloud/deploy/terraform/"
+localTerraformAppDir: "{{ dirTerraform }}/.terraform/modules/orc8r-app/orc8r/cloud/deploy/terraform/"
+
+orc8rSource: "github.com/magma/magma//orc8r/cloud/deploy/terraform/orc8r-aws?ref={{ orc8rVersion }}"
+orc8rAppSource: "github.com/magma/magma//orc8r/cloud/deploy/terraform/orc8r-helm-aws?ref={{ orc8rVersion }}"
+
+#Terraform variables
+#
+
+nmsDbPassword: testpassword
```

Comment:
Is this dummy or do we set the password as "testpassword"

Reply:
Right now the value set in here (in this case "testpassword") is what is passed on to main.tf. This should ideally be moved to secrets.yaml so that code, configuration information and authentication information can be separated. I will create a PR for this and update.

Reply:
https://github.com/magma/magma/issues/5110